scripts/proguard.pro

```diff
@@ -31,6 +31,11 @@
 # need to keep mixin names
 -keep class baritone.launch.** { *; }
 
+#try to keep usage of schematica in separate classes
+-keep class baritone.utils.schematic.schematica.*
```

Comment:
use ** so that you don't get memed by making a subpackage and wondering why proguard not keeping it